.gitignore

```diff
@@ -1,3 +1,3 @@
 # Ignore .DS_Store 
 .DS_Store
-#ga/*/*/resources/*
+##ga/*/*/resources/*
```

Comment:
This change wouldn't have an effect as the line has already been commented out. I believe Wendy was saying that this line shouldn't be commented out. It'll be good to sort it out.